docs/plans/company-news-opensearch-search.md

```
@@ -0,0 +1,307 @@
+        size: 반환할 결과 개수
+        published_after: 이 날짜 이후의 뉴스만 검색
+        search_fields: 검색할 필드 목록 (기본: ["title", "content"])
+
+    Returns:
+        list: 검색 결과 (news_id, title, score, published_at)
+    """
+```
+
+#### 검색 전략 옵션
+
+**A. 텍스트 매칭 (단순)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)